packages/xstate-test/test/paths.test.ts

```diff
@@ -172,15 +175,17 @@ describe('transition coverage', () => {
     const model = createTestModel(machine);
 
     const paths = model.getShortestPaths({
-      eventCases: {
-        NEXT: [{ value: 0 }, { value: 100 }, { value: 1000 }]
-      }
+      events: [
+        { type: 'NEXT', value: 0 },
+        { type: 'NEXT', value: 100 },
+        { type: 'NEXT', value: 1000 }
+      ]
     });
 
     // { value: 1000 } already covered by first guarded transition
     expect(paths.map((path) => path.description)).toMatchInlineSnapshot(`
       Array [
-        "Reaches state \\"b\\": NEXT ({\\"value\\":0})",
+        "Reaches state \\"b\\": NEXT ({\\"value\\":0}) → NEXT ({\\"value\\":0})",
```

Comment:
is this an intentional change? a bug that was fixed perhaps?

Reply:
Updated snapshots